.github/workflows/audit-frontend.yaml

```diff
@@ -0,0 +1,40 @@
+# GitHub action script to audit for unused frontend packages.
+name: Audit Frontend
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+      - ready_for_review
+
+jobs:
+  audit-core-web: # Audit for unused frontend packages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: depcheck
+        uses: tj-actions/depcheck@v8
+        with:
+            dir: services/core-web
+            ignores: 'eslint,babel-*'
+        continue-on-error: true
+
+  audit-minespace: # Audit for unused frontend packages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: depcheck
+        uses: tj-actions/depcheck@v8
+        with:
+            dir: services/minespace-web
+            ignores: 'eslint,babel-*'
+        continue-on-error: true
+
```

Comment:
Can we print this to the PR, without failing the checks, this might nudge the devs to work towards removing the unwanted deps gradually